js/src/analysis/isWordFormRecognitionActive.js

```diff
@@ -0,0 +1,16 @@
+var getL10nObject = require( "./getL10nObject" );
+
+var isUndefined = require( "lodash/isUndefined" );
+
+/**
+ * Returns whether or not the content analysis is active
+ *
+ * @returns {boolean} Whether or not the content analysis is active.
```

Comment:
I guess this docblock should refer to "word forms analysis" and not to the content analysis. Minor: missing final period on line 6.